modules/compiler/utils/source/barrier_regions.cpp

```diff
@@ -972,6 +972,9 @@ void compiler::utils::Barrier::MakeLiveVariableMemType() {
     // so record this and the matching byte offset into the struct.
 #if LLVM_VERSION_GREATER_EQUAL(18, 0)
     auto DbgIntrinsics = findDbgDeclares(member.value);
+#if LLVM_VERSION_GREATER_EQUAL(19, 0)
```

Comment:
Nitpick, but would it be possible to move the DPVDeclares outside of the `#if LLVM_VERSION_GREATER_EQUAL(18, 0)`? The nesting of the ifdefs makes it a bit hard to skim.

Reply:
Yeah, we don't need it here yet, I've moved it below.